pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -826,13 +826,31 @@ cdef _to_py_int_float(v):
 cdef class _Timedelta(timedelta):
     # cdef readonly:
     #    int64_t value      # nanoseconds
-    #    object freq        # frequency reference
-    #    bint is_populated  # are my components populated
+    #    bint _is_populated  # are my components populated
     #    int64_t _d, _h, _m, _s, _ms, _us, _ns
 
     # higher than np.ndarray and np.matrix
     __array_priority__ = 100
 
+    @property
+    def freq(self):
+        # GH#46430
+        warnings.warn(
```

Comment:
what does this do if its called? return None?

Reply:
correct

Reply:
can you make that explcit (and add the annotation)